src/wpool_queue_manager.erl

```diff
@@ -106,14 +121,17 @@ sync_send_event_to_available_worker(QueueManager, Event, Timeout) ->
 -spec sync_send_all_event_to_available_worker(queue_mgr(), any(), timeout()) ->
         noproc | timeout | atom().
 sync_send_all_event_to_available_worker(QueueManager, Event, Timeout) ->
-  Expires =
-    case Timeout of
-      infinity -> infinity;
-      Timeout -> now_in_microseconds() + Timeout * 1000
-    end,
+  Expires = expires(Timeout),
   try
     gen_server:call(
       QueueManager, {sync_all_event_available_worker, Event, Expires}, Timeout)
+  of
+    {'EXIT', _, noproc} ->
+      noproc;
+    {'EXIT', Worker, Exit} ->
+      exit({Exit, {gen_fsm, sync_send_all_state_event, [Worker, Event, Timeout]}});
```

Comment:
Line 132 is too long:       exit({Exit, {gen_fsm, sync_send_all_state_event, [Worker, Event, Timeout]}});.